hc/front/tests/test_add_channel.py

```diff
@@ -38,4 +38,30 @@ def test_instructions_work(self):
             self.assertContains(r, "Integration Settings", status_code=200)
 
     ### Test that the team access works
+    def test_teamaccess_works(self):
+        url = "/integrations/add/"
+        form = {"kind": "email", "value": "alice@example.com"}
+
+        self.client.login(username="alice@example.org", password="password")
+        self.client.post(url, form)
+        alice_channels = Channel.objects.filter(user=self.alice)
+        self.client.logout()
+
+        self.client.login(username="bob@example.org", password="password")
+        r = self.client.get("/integrations/")
```

Comment:
use `reverse`
You can read about it from the Django documentation

Reply:
Noted.